### PR TITLE
DOCS-12795 test edit

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-12795 test edit- more info if needed

### CODE REVIEW

File: modules/ROOT/pages/to-search-with-graph-api.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ Search Exchange assets for a string value by using the Exchange Graph API. You c
 
 The Exchange Graph API accesses Anypoint Exchange and Anypoint Platform, and a search requires authentication to access assets in a private Exchange. If you only search for MuleSoft public assets, you don't need authentication.
 
-The Exchange Graph API can be used without an Anypoint Platform account to find assets made publicly available by MuleSoft. Accessing private content through the API requires an Anypoint Platform account.
+Use the Exchange Graph API without an Anypoint Platform account to find assets made publicly available by MuleSoft. Accessing private content through the API requires an Anypoint Platform account.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/to-search-with-graph-api.adoc:11]: this is incorrect, instead of "103" it should say "301"

Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-search-with-graph-api.adoc:11]: thanks! I fixed this

Comment (wwarner-salesforce) [inline modules/ROOT/pages/to-search-with-graph-api.adoc:11]:
```suggestion
Use the Exchange Graph API without a fancy Anypoint Platform account to find assets made publicly available by MuleSoft. Accessing private content through the API requires an Anypoint Platform account.
```

File: modules/ROOT/pages/to-search-with-graph-api.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ Search Exchange assets for a string value by using the Exchange Graph API. You c
 
 The Exchange Graph API accesses Anypoint Exchange and Anypoint Platform, and a search requires authentication to access assets in a private Exchange. If you only search for MuleSoft public assets, you don't need authentication.
 
-The Exchange Graph API can be used without an Anypoint Platform account to find assets made publicly available by MuleSoft. Accessing private content through the API requires an Anypoint Platform account.
+Use the Exchange Graph API without a fancy Anypoint Platform account to find assets made publicly available by MuleSoft. Accessing private content through the API requires an Anypoint Platform account.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/to-search-with-graph-api.adoc]:
```suggestion
Use the Exchange Graph API without an Anypoint Platform account to find assets made publicly available by MuleSoft. Accessing private content through the API requires an Anypoint Platform account.
```

Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-search-with-graph-api.adoc]: Fixed. Thanks!

**Review by JennyHajee — APPROVED** [review]

**Pull request closed without merging**